transactions/move-admin-resources/may-2023/move-admin-resources.cdc

```diff
@@ -0,0 +1,72 @@
+import FlowIDTableStaking from 0x8624b52f9ddcd04a
+import FlowEpoch from 0x8624b52f9ddcd04a
+import FlowClusterQC from 0x8624b52f9ddcd04a
+import FlowDKG from 0x8624b52f9ddcd04a
+
+/// Transaction to move epoch admin resources from the staking
+/// account to the service account
+
+transaction() {
+
+    prepare(stakingAccount: AuthAccount, serviceAccount: AuthAccount) {
+
+        // move the staking admin to the service account
+        let stakingAdmin <- stakingAccount.load<@FlowIDTableStaking.Admin>(from: FlowIDTableStaking.StakingAdminStoragePath)
+            ?? panic("Could not load staking admin from staking account")
+
+        // The service account already stores a staking admin capability, so load that first
+        serviceAccount.load<Capability>(from: FlowIDTableStaking.StakingAdminStoragePath)
+            ?? panic("Could not load staking admin capability from the service account")
```

Comment:
Does this part only check something? It doesn't set anything

Reply:
On mainnet, there is a capability stored in the staking admin storage path, so since we want to store the actual admin there, we need to remove and the delete the existing capability first